shrine/lib/login.dart

```diff
@@ -58,32 +58,36 @@ class _LoginPageState extends State<LoginPage> {
               color: kShrineBrown900,
               child: TextField(
                 controller: _passwordController,
-                decoration: InputDecoration(
+                decoration: const InputDecoration(
                   labelText: 'Password',
                 ),
               ),
             ),
-            ButtonBar(
+            Wrap(
```

Comment:
What does this change do?

Reply:
Hmm. This was probably introduced for the flutter/flutter version of Shrine, which has an extra button (exit).  It's not serving much of a purpose here, other than keeping the code in sync with the other copy.

I can take it out.

Reply:
That exit button don't look good anyway. Should be a back button at the top.

Reply:
In its defense, it was hacked into the code at the last minute by an engineer with no eye for the visual. 😄